### ISSUE TITLE
Use of RStudio Cloud workspaces for NHS-R Community Intro to R and R Studio workshop

### ISSUE THREAD

**Issue by send2dan** [Lextuga007/Presentations#8]
Hello,

Emma and I thought it might be useful to incl information on how to add new members to the Rstudio Cloud workspace? 

e.g. go to ... and click 'add member' and then add email address. This will send a link to the recipient

**Comment by Lextuga007** [comment]
Just so you know I don't publish the invitation link publicly as it is an open link to the workspace and could be used by people for things other than training.

Did you mean to post this on the [NHS-R Community intro page](https://github.com/nhs-r-community/intro_r/issues)?

**Comment by send2dan** [comment]
OK, understood RE: don't publish the invitation link publicly

I added this comment to 'train the trainers' as adding new members to the workspace was necessary for two participants at today's intro session :)

**Comment by Lextuga007** [comment]
It's an interesting thing about the Cloud though as I've wondered how useful it is to have a workspace set up like this. I've also used code in courses like https://philosopher-analyst.netlify.app/rmd-workshop/workshop-rmd/prework/ where people load the packages and data (using {usethis}) into their own workspaces which works really well, even if they are really new to R. There is a lot of admin related to adding people to the workspace as suggested, or even sending people's work to their own workspace after the course but NHS-R Community have a paid subscription to R Studio Cloud so I'm not sure what's best. I'll mention this in the Slack group to see what people's thoughts are on the workspaces.

**Comment by send2dan** [comment]
Another issue with the RStudio Cloud is that it looks like, if I log into another participant's project, I kick them out of it? (i.e. you can't have two users in the same project simultaneously?)

**Comment by Lextuga007** [comment]
Yes, that's true - caused me a lot of trouble as I have admin rights so when I was supporting someone else I couldn't create my own space. 

My solution of having people use their own workspaces means that doesn't happen but also that you can't see their work through the cloud, only if they share; I'm not sure if that's a good or a bad thing.
